src/Layouts/PaginationLayout.js

```diff
@@ -0,0 +1,8 @@
+import { Outlet } from "react-router-dom";
+export function PaginationLayout() {
+  return (
+    <>
```

Comment:
yup, I was thinking about giving a page with pagination instead of a layout. But `number` is a {param} in the ListPage and I don't know how it would work, I haven't checked if it's possible to play with the path like that

Reply:
i mean `path="list/:number" (number changes depending on the page)